src/main.rs

```diff
@@ -1,3 +1,20 @@
+use crate::types::particle::Particle;
+use crate::physics::gravity::gravitational_force;
+
+pub mod physics;
+pub mod types;
+
 fn main() {
-    println!("Hello, world!");
+    let line = "-".repeat(150);
+    let body1: Particle = Particle::new(0, [-50.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0], 10.5e14);
+    let body2: Particle = Particle::new(0, [50.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0], 1.5e14);
+
+    println!("First Body: {:?}", body1);
+    println!("Second Body: {:?}", body2);
+    println!("{}", line);
+
+    let force_on_first_body = gravitational_force(&body1, &body2);
+
+    println!("Force on first object = {:?}", force_on_first_body);
+    println!("Force on second object = {:?}", vecmath::vec3_neg(force_on_first_body));
```

Comment:
Maybe we can move this into separate functions?

```
fn display_particle_info(label: &str, particle: &Particle) {
    println!("{}: {:?}", label, particle);
}

fn display_force(label: &str, force: &[f64; 3]) {
    println!("{} = {:?}", label, force);
}
```
```suggestion
    display_particle_info("First Body", &body1);
    display_particle_info("Second Body", &body2);

    println!("{}", line);

    let force_on_first_body = gravitational_force(&body1, &body2);
    let force_on_second_body = vecmath::vec3_neg(force_on_first_body);

    display_force("Force on first object", &force_on_first_body);
    display_force("Force on second object", &force_on_second_body);
```

Reply:
Thank you for the suggestion, but this is a temporary solution for demonstrating intermediate functionality. This code will be removed in future patches, so creating separate functions is unnecessary at this point.